commands/command_pull.go

```diff
@@ -113,7 +113,8 @@ func pull(filter *filepathfilter.Filter) {
 	}
 
 	if singleCheckout.Skip() {
-		fmt.Println(tr.Tr.Get("Skipping object checkout, Git LFS is not installed."))
+		fmt.Println(tr.Tr.Get("Skipping object checkout, Git LFS is not installed for this repository.\n" +
+		                      "Consider installing it with 'git lfs install'."))
```

Comment:
I think one of these lines has trailing whitespace, which is causing CI to fail.

While you're making that change, I think it will be easier for the code which extracts strings for translation if we leave it as one long string on one line, just with a newline in the middle.

Reply:
Easy. I amended the snippet to just one line with the newline character. Sorry it took three reviews for a one-line string. :)